core/modules/views_ui/css/views_ui.admin.theme.css

```diff
@@ -232,6 +271,46 @@ input.form-radio {
   font-weight: bold;
 }
 
+@media (max-width: 768px) {
+  table.sticky-header {
+    display: none;
+  }
+
+  [id="views-list"] thead {
```

Comment:
I'm guessing you use this selector instead of `#views-list` for specificity reasons? Maybe we should modify the markup to add a class so that this unusual selector could be avoided.